### ISSUE TITLE
Play around with tt-train

### ISSUE THREAD

**Issue by vladimirjovanovicTT** [tenstorrent/tt-blacksmith#5]
See what TTNN ops they are targeting.
I remember tt-folks telling me they are using bfloat16 and 4 dimensions for all tensors.
We should take a look at how they handle batch norm and dropout (although this should be less of an issue for them).
